src/components/Vqb.vue

```diff
@@ -46,16 +50,22 @@ export default class Vqb extends Vue {
   @Mutation setPipeline!: (payload: Pick<VQBState, 'pipeline'>) => void;
   @Mutation toggleStepEdition!: () => void;
 
+  formToInstantiate?: VueConstructor<Vue>;
   initialValue: any = undefined;
   editedStepIndex: number = -1;
+  stepName!: string;
 
   get isStepCreation() {
     return this.editedStepIndex === -1;
   }
 
-  openStepForm(params: any, index: number) {
-    // params.name will be used to choose the right form
+  openStepForm(params: PipelineStep, index: number) {
     // after that, we delete from params to pass down the others keys to initialValue
+    this.formToInstantiate = STEPFORM_REGISTRY[params.name];
+    if (this.formToInstantiate === undefined) {
```

Comment:
As we will not have for a moment all the functions available, we should have a real UI to inform when a function is not available yet